presentation/src/main/java/com/example/user/test/presentation/base/BaseNavigator.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.user.test.presentation.base
+
+import android.app.Activity
+import android.content.Intent
+import android.support.v4.app.FragmentActivity
+import android.widget.Toast
+import com.example.user.test.presentation.flow.FinishFlow
+import com.example.user.test.presentation.flow.StartFlow
+import ru.terrakok.cicerone.android.SupportFragmentNavigator
+import ru.terrakok.cicerone.commands.Command
+
+abstract class BaseNavigator(val activity: FragmentActivity, containerId: Int) :
```

Comment:
Refactor this logic as it done in `Cicerone` sample and combine it with `GitFox` implementation, provided by the link to you.